ikarus/utils/fancyActivationFunctions.hh

```diff
@@ -0,0 +1,48 @@
+// SPDX-FileCopyrightText: 2022 The Ikarus Developers mueller@ibb.uni-stuttgart.de
+// SPDX-License-Identifier: LGPL-3.0-or-later
+
+#pragma once
+#include <memory>
+#include <optional>
+
+#include <dune/common/float_cmp.hh>
+
+#include <ikarus/solver/linearSolver/linearSolver.hh>
+
+namespace Ikarus::AdaptiveStepSizing {
+
+  struct ActivationFunctionArgs {
+    double r_0{0.01};
+    double r_i{0.05};
```

Comment:
why are here default but for the functions below, there is no default argument. Is it reasonable to use these defaults?